src/Insect/Environment.purs

```diff
@@ -130,6 +147,9 @@ initialEnvironment =
         , constFuncN "maximum" (lmap QConversionError <<< Q.max <<< NonEmptyList)
         , constFuncN "mean" (lmap QConversionError <<< Q.mean <<< NonEmptyList)
         , constFunc  "round" Q.round
+        , constFunc  "secant" Q.sec
+        , constFunc  "sech" Q.sech
```

Comment:
I opted to keep `sech` and `asech`, as neither conflicts with any other identifiers, but you may want to remove them because there's no `sec` (interpreted as second) or `asec` (interpreted as attosecond). Personally, I think keeping the shortened names is okay.

Reply:
I agree.